Tools/Scripts/libraries/webkitbugspy/webkitbugspy/issue.py

```diff
@@ -123,6 +124,15 @@ def description(self):
             self.tracker.populate(self, 'description')
         return self._description
 
+    @property
+    def state(self):
+        if self._state is None:
+            self.tracker.populate(self, 'state')
+        return self._state
+
+    def set_state(self, state):
+        return bool(self.tracker.set(self, state=state))
```

Comment:
Can we include a test for the other two issue trackers to make sure we behave sanely when `state` isn't supported?